content/programming-cookbook/tips-and-tricks/json-to-csharp-classes.md

```diff
@@ -0,0 +1,39 @@
+---
+title: JSON to C# Classes
+showMetadata: true
+editable: true
+showToc: true
+---
+
+# Visual Studio
+
+- Example JSON data
+```json
+{
+    "id": 1,
+    "name": "Peter",
+    "lastname": "Parker"
+}
+```
+
+- Create your .cs class
+- Copy your JSON data
+- Select Edit => Paste Special => Paste JSON As Classes
```

Comment:
These three bullets is an ordered list. I suggest you to add small heading line before them for example

```md
## Steps
```
and use ordered list instead of unordered list for example:

```md
1. Create your .cs class
2. Copy ..
3. Select ...
```

Reply:
@surrealist Good comment.  It is a steps list so I agree to use an ordered list.

But for me, I get used to an unorder list. 
The good thing about it is that we don't need to keep updating any number if we add a new step in a middle or rearrange an order.

For a person who read this list, I still think they can interpret this steps list without having order number.

Anyway, I agree to update it to an ordered list but just only point out what I have concerned.

Reply:
Thanks. You can use **only number 1** for ordered list if you concerned that.  The numbers will running automatically.
 
For this example markdown:
```md
1. step one
1. step two
1. step three
```

will generate result as
1. step one
1. step two
1. step three